src/components/survey/generic/container.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import style from "./style.scss";
+
+export default function Container({children, progress}) {
+  return (
+    <div className={`${style.container}`}>
```

Comment:
Doesn't need the string interpolation here:
`    <div className={style.container}>`